components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml

```diff
@@ -149,6 +154,8 @@
 <config_Redi_horizontal_taper>'ramp'</config_Redi_horizontal_taper>
 <config_Redi_horizontal_taper ocn_grid="SOwISC12to60E2r4">'RossbyRadius'</config_Redi_horizontal_taper>
 <config_Redi_horizontal_taper ocn_grid="ECwISC30to60E2r1">'RossbyRadius'</config_Redi_horizontal_taper>
+<!-- To do: ramp for WC but RossbyRadius for Cryo -->
+<config_Redi_horizontal_taper ocn_grid="ECwISC30to60E3r1">'ramp'</config_Redi_horizontal_taper>
```

Comment:
@jonbob and @darincomeau, these are the situations that need discussion before this goes into E3SM.  We want to use the `ECwISC30to60E3r1` (or maybe the `IcoswISC30E3r1` if results look better) for both WCYCL and CRYO configurations but there's currently not a way to have different parameter values for the same mesh.  How can we handle this?